common/src/main/java/com/wynntils/core/features/FeatureRegistry.java

```diff
@@ -183,6 +184,7 @@ public static void init() {
         registerFeature(new PreventTradesDuelsFeature());
         registerFeature(new QuestInfoOverlayFeature());
         registerFeature(new QuickCastFeature());
+        registerFeature(new ChatRedirectFeature());
```

Comment:
You forgot to maintain alphabetical order

Reply:
..! 🙄 Oh dead. I named it `RedirectFeature` originally, and placed it in the right spot. Then I used IntelliJ renaming and forgot about it...

I also noticed that I had managed to miss pushing my last updates from yesterday, which -- ironically enough -- was about alphabetic order as well. :-)